environment/environment.go

```diff
@@ -188,7 +188,7 @@ func (e *Environment) Prepare() error {
 
 	if len(metaFiles) == 0 {
```

Comment:
as we eventually export metadata, I don't think this step is necessary anymore.
we may be able to remove this block, and to wrap the local metadata application step with
```
if len(metaFiles) != 0 {
}
```

Reply:
I think we can count on metadata to be there. I've just removed the metadata export step for now.